src/Latte/Runtime/FilterExecutor.php

```diff
@@ -178,6 +178,7 @@ private function prepareFilter(string $name): array
 				? new \ReflectionMethod($callback[0], $callback[1])
 				: new \ReflectionFunction($callback);
 			$this->_static[$name][1] = ($tmp = $ref->getParameters())
+				&& $tmp[0]->getType() !== null
```

Comment:
Btw, did you find why is it null at PHP 7.4?

Reply:
I think that all tests failed, before getName function, it was casted to string, so it could not be null in if check